### PR TITLE
Paas 2022 dd apm j customer

### PR DESCRIPTION
**Pull request by adoux**
JIRA issue: https://jira.jahia.org/browse/PAAS-2022

Short description:

### CODE REVIEW

File: mixins/jcustomer.yml [hunk 1]
```diff
@@ -350,4 +350,4 @@ actions:
     - cmd [cp]: |-
         setenv=$(find /opt/jcustomer/jcustomer/bin -name setenv)
         echo 'APM_OPTS="-javaagent:'${JCUSTOMER_HOME}'/jcustomer/datadog/dd-java-agent.jar -Ddd.logs.injection=true -Ddd.service=jcustomer -Ddd.env='${env.envName}' -Ddd.profiling.enabled=true"' >> $setenv
-        echo ${DATADOG_APM_ENABLED:-true}' && JAVA_OPTS+=" $APM_OPTS"' >> $setenv
+        ${DATADOG_APM_ENABLED:-true} && echo 'JAVA_OPTS+=" $APM_OPTS"' >> $setenv
```
Comment (kevinvandel) [inline mixins/jcustomer.yml:353]:
```suggestion
        echo '${DATADOG_APM_ENABLED:-true} && JAVA_OPTS+=" $APM_OPTS"' >> $setenv
```

File: packages/jcustomer/migrations/migrate-to-v14.yaml [hunk 2]
```diff
@@ -51,7 +51,7 @@ actions:
         setenv=$(find /opt/jcustomer/jcustomer/bin -name setenv)
         if ! grep -q "APM_OPTS" $setenv; then
           echo 'APM_OPTS="-javaagent:'${JCUSTOMER_HOME}'/jcustomer/datadog/dd-java-agent.jar -Ddd.logs.injection=true -Ddd.service=jcustomer -Ddd.env='${env.envName}' -Ddd.profiling.enabled=true"' >> $setenv
-          echo ${DATADOG_APM_ENABLED:-true}' && JAVA_OPTS+=" $APM_OPTS"' >> $setenv
+          ${DATADOG_APM_ENABLED:-true} && echo 'JAVA_OPTS+=" $APM_OPTS' >> $setenv
```
Comment (kevinvandel) [inline packages/jcustomer/migrations/migrate-to-v14.yaml:54]:
```suggestion
          echo '${DATADOG_APM_ENABLED:-true} && JAVA_OPTS+=" $APM_OPTS' >> $setenv
```

**Review by kevinvandel — CHANGES_REQUESTED** [review]

File: mixins/jcustomer.yml (hunk 1 above)
Comment (adoux) [reply mixins/jcustomer.yml]: stupid me :man_facepalming: 
Fixed

File: packages/jcustomer/migrations/migrate-to-v14.yaml (hunk 2 above)
Comment (adoux) [reply packages/jcustomer/migrations/migrate-to-v14.yaml]: Fixed

**Review by kevinvandel — APPROVED** [review]

**Pull request merged**